Dockerfile

```diff
@@ -327,34 +329,6 @@ RUN if [[ -f /docker-context-files/requirements.txt ]]; then \
         pip install --no-cache-dir --user -r /docker-context-files/requirements.txt; \
     fi
 
```

Comment:
BTW. We do not need those labels any more because we are not pusing the 'build' segment to our repo any more. This yet another great feature of buildkit cache, that you can built it with "mode=max" where the cache will contain layers for all segments - the final and intermediate ones.